internal/gitlab/testdata/changed_line/all_lines_changed/formatted.yaml

```diff
@@ -0,0 +1,2 @@
+x: y
+z: w
```

Comment:
In a realistic scenario, the two things that are being compared for codequality will both end with new lines. All the testdata files should have new lines at the end of the files.